backend/run/.gitignore

```
@@ -7,11 +7,16 @@
 
 # Except for the static folder
 !static
-!static/.gitkeep
 # But its content
 static/*
+!static/.gitkeep
+
 # Except for the media folder
 !media
-!media/.gitkeep
 # But its content
 media/*
+!media/.gitkeep
+
+!log
+log/*
+!log/.gitkeep
```

Git tells me it wants a newline at the end of this file
```suggestion
!log/.gitkeep

```